code/game/objects/structures/false_walls.dm

```diff
@@ -11,7 +11,7 @@
 * around us, then checks the difference.
 */
 /proc/getOPressureDifferential(var/turf/loc)
-	var/minp=16777216;
+	var/minp=16777216; //What is even the significance of this number?
```

Comment:
We can just
You know
Change the number in the line that all these comments are on into the actual max pressure
I'm mostly sure that wouldn't break anything